entities/storobj/storage_object.go

```diff
@@ -325,14 +374,13 @@ func SearchResults(in []*Object, additional additional.Properties) search.Result
 	return out
 }
 
-func SearchResultsWithDists(in []*Object, additional additional.Properties,
+func SearchResultsWithDists(in []*Object, addl additional.Properties,
```

Comment:
I don't see the benefit of doing this

Reply:
The original name collides with the imported package and broke compilation, so I went with this common abbreviation  instead